partiql-parser/src/parse/partiql.lalrpop

```diff
@@ -1093,6 +1104,12 @@ LiteralNumber: ast::Lit = {
 LiteralIon: ast::Lit = {
     <ion:"Ion"> => ast::Lit::IonStringLit(ion.to_owned()),
 }
+#[inline]
+LiteralDateTime: ast::Lit = {
+    <ion:"DATE"> <s:"String"> => ast::Lit::DateTimeLit(ast::DateTimeLit::DateLit(s.to_owned())),
```

Comment:
I think the `<ion:` can be removed from all of these.

Reply:
I agree, done.